tests/unit_tests/docmaps/provider_test.py

```diff
@@ -38,23 +38,44 @@
 PREPRINT_LINK_PREFIX = 'https://test-preprints/'
 PREPRINT_LINK_1_PREFIX = f'{PREPRINT_LINK_PREFIX}{DOI_1}'
 PREPRINT_LINK_1 = f'{PREPRINT_LINK_1_PREFIX}v{PREPRINT_VERSION_1}'
+PREPRINT_LINK_2_PREFIX = f'{PREPRINT_LINK_PREFIX}{DOI_2}'
+PREPRINT_LINK_2 = f'{PREPRINT_LINK_2_PREFIX}v{PREPRINT_VERSION_2}'
 
+ELIFE_DOI_VERSION_STR_1 = 'elife_doi_version_str_1'
+ELIFE_DOI_VERSION_STR_2 = 'elife_doi_version_str_2'
+
+TDM_PATH_1 = 'tdm_path_1'
+TDM_PATH_2 = 'tdm_path_2'
+
+PREPRINT_DETAILS_1 = {
+    'preprint_url': PREPRINT_LINK_1,
+    'elife_doi_version_str': ELIFE_DOI_VERSION_STR_1,
+    'preprint_doi': DOI_1,
+    'preprint_version': PREPRINT_VERSION_1,
+    'preprint_published_at_date': date.fromisoformat('2021-01-01'),
+    'tdm_path': TDM_PATH_1
+}
+
+PREPRINT_DETAILS_2 = {
```

Comment:
There seem to be a test missing using `PREPRINT_DETAILS_2`.

Reply:
I haven't add the test for this yet because this PR just for not to update the code, only for handling the structural changes in BQ result. If that makes sense. Assuming only one preprint_doi will be there.